### ISSUE TITLE
cljunit doesn't pick up tests with maven

### ISSUE THREAD

**Issue by kurt-o-sys** [mikera/cljunit#4]
Using eclipse: 'Run As -> JUnit Tests' runs as expected, showing 1 failed test in the JUnit tab.
However: 'Run As -> Maven test' (or 'Run As -> Maven install') does not. My java test class (PropertyTests extends ClojureTest) is picked up, but no tests are found:

```
Running myproject.PropertyTests
Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.828 sec

Results :

Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
```

Some code, see below. I tried with and without filter, both giving the same results.

Why are the tests not picked up with maven, although they are when running the tests in eclipse with 'Run As -> JUnit Tests'?

---

src/test/java/myproject/PropertyTests.java:

```
public class PropertyTests extends ClojureTest {
    /** {@inheritDoc} */
    @Override
    public String filter() {
        return "myproject";
    }
}
```

---

src/test/clojure/myproject/gentest.clj:

```
(ns myproject.gentest
  (:require [clojure.test :refer :all])

(deftest fixme
  (testing "FIXME, I fail."
           (is (= 1 0))))
```

---

pom.xml:

```
    <depencendies>
    [...]
        <dependency>
            <groupId>junit</groupId>
            <artifactId>junit</artifactId>
            <version>4.12</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>org.clojure</groupId>
            <artifactId>clojure</artifactId>
            <version>1.8.0</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>net.mikera</groupId>
            <artifactId>cljunit</artifactId>
            <version>0.4.0</version>
            <scope>test</scope>
        </dependency>
        [...]
    </dependencies>
    <build>
        <testResources>
            <testResource>
                <directory>src/test/clojure</directory>
            </testResource>
        </testResources>
        <plugins>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-surefire-plugin</artifactId>
                <configuration>
                    <includes>
                        <include>**/*Test.java</include>
                        <include>**/*Tests.java</include>
                    </includes>
                </configuration>
            </plugin>
        </plugins>
    </build>
```

**Comment by mikera** [comment]
Hmmm not sure about this. May need some further investigation.

**Comment by kurt-o-sys** [comment]
Any ideas what to try/test?

**Comment by mikera** [comment]
My suspicion is that the Java files aren't being compiled by Maven? i.e. your `PropertyTests` class isn't on the classpath when Maven runs the test suite?

**Comment by mikera** [comment]
Fixed in 0.6.0
